src/constants.ts

```diff
@@ -46,7 +46,7 @@ export const foaf = {
 
 /** @internal */
 export const ldp = {
-  inbox: "https://www.w3.org/ns/ldp#inbox",
+  inbox: "http://www.w3.org/ns/ldp#inbox",
```

Comment:
Can't really blame you, the problem lies in that we're not using auto-generated constants yet :)

Reply:
(Although AS is extra problematic because it doesn't, as far as I'm aware, expose the vocabulary as RDF, so `rdf-namespaces` doesn't include it either.)

Reply:
You're right, the canonical AS vocabulary is published in JSON-LD, which was also a head scratcher for the vocab constants generator.